devicetypes/smartthings/orvibo-gas-detector.src/Orvibo-Gas-detector.groovy

```diff
@@ -66,8 +66,20 @@ def parse(String description) {
 		log.debug "enroll response: ${cmds}"
 		result = cmds?.collect { new physicalgraph.device.HubAction(it)}
 	}
+
 	return result
 }
+
+def parseAttrMessage(String description){
+	def descMap = zigbee.parseDescriptionAsMap(description)
+	def map = [:]
+	if (descMap?.clusterInt == 0x0500 && descMap.attrInt == 0x0002) {
```

Comment:
please use `zigbee.IAS_ZONE_CLUSTER` and `zigbee.ATTRIBUTE_IAS_ZONE_STATUS` as you did in `refresh()`

Reply:
Ok,I will change this.